docker/nginx-gateway-k8s.conf.template

```diff
@@ -70,6 +70,13 @@ server {
         internal;
     }
 
+    location = /logout {
```

Comment:
So `/logout` was chosen simply because it was free. The `Clear-Site-Data` header is a server response so needed a concrete endpoint for the client to hit in order for the server to send the header. So as long as `/auth/logout` is not taken up by anything else then that should be fine.